tiledb/sm/misc/test/compile_math_main.cc

```diff
@@ -30,5 +30,6 @@
 
 int main() {
   (void)tiledb::sm::utils::math::ceil(1, 1);
+  (void)tiledb::sm::utils::math::log(1, 1);
```

Comment:
Just curious - why the change?

Reply:
Artifact from trying to fix link issues... removed.